docs/installation.md

```diff
@@ -87,6 +87,12 @@ The following table lists the configurable parameters of the OpenEBS chart and t
 
 Specify each parameter using the `--set key=value` argument to `helm install`.
 
+If you want to change the default values, use — set key=value argument in the installation command as in the following example.
```

Comment:
If you want to change the default values, use  -- set key=value argument in the installation command as in the following example.